Basic-Car-Maintenance/Shared/Odometer/Views/OdometerView.swift

```diff
@@ -21,37 +21,29 @@ struct OdometerView: View {
         NavigationStack {
             List {
                 ForEach(viewModel.readings) { reading in
-                    VStack(alignment: .leading, spacing: 8) {
-                        Text("\(reading.distance) \(reading.isMetric ? "km" : "mi")")
-                            .font(.title3)
-                        
-                        let vehicleName = viewModel.vehicles.first { $0.id == reading.vehicleID }?.name
-                        if let vehicleName {
-                            Text("For \(vehicleName)")
-                        }
-                        
-                        Text("\(reading.date.formatted(date: .abbreviated, time: .omitted))")
-                    }
-                    .swipeActions(edge: .trailing, allowsFullSwipe: true) {
-                        Button(role: .destructive) {
-                            Task {
-                                await viewModel.deleteReading(reading)
+                    let vehicleName = viewModel.vehicles.first { $0.id == reading.vehicleID }?.name
+                    ReadingView(odometerReading: reading, vehicleName: vehicleName)
+                        .listRowInsets(.init(top: 16, leading: 24, bottom: 16, trailing: 24))
```

Comment:
I think it's fine without these insets, I don't think it makes a dramatic difference
I fixed this